src/settings/index.ts

```diff
@@ -9,6 +9,8 @@ export interface DeploymentSettings {
 export interface Settings {
   /** Where do we connect to */
   readonly backend: BackendSettings;
+  /** Are we using Stargate (Launchpad = false) */
+  readonly stargateEnabled: boolean;
```

Comment:
Can you move this to `BackendSettings`? For every given backend, it is either launchpad or stargate